source/common/http/http3/conn_pool.h

```diff
@@ -156,7 +158,7 @@ class Http3ConnPoolImpl : public FixedHttpConnPoolImpl {
 
   // Store quic helpers which can be shared between connections and must live
   // beyond the lifetime of individual connections.
-  std::unique_ptr<Quic::PersistentQuicInfoImpl> quic_info_;
+  Quic::PersistentQuicInfoImpl& quic_info_;
```

Comment:
I would be inclined to split out the moving of the PersistentQuicInfo object changes from the "enable 0-RTT" changes.

Reply:
I'm fine just moving it and relying on existing tests for the move, to keep the move separate from the functional change.

Reply:
reverted 0-RTT related changes

Reply:
nit: Since the 0-RTT changes have been reverted, can you edit the PR description to avoid mentioning them?

Reply:
done